src/core/bittorrent/session.cpp

```diff
@@ -2111,6 +2122,9 @@ void Session::handleAddTorrentAlert(libtorrent::add_torrent_alert *p)
             }
         }
 
+        if (pref->isAddTrackersEnabled() && !torrent->isPrivate())
+            torrent->addTrackers(m_additionalTrackers);
```

Comment:
I think it is a better idea to do this inside `Session::addTorrent_impl()` by using `torrent_info::add_tracker()`
Of course you'll need to extend our TorrentInfo and remove the const from addTorrent_impl().
@glassez do you agree with this?
However, this probably won't work for magnet links. And the docs about `add_torrent_params::trackers` say:
>trackers
    If the torrent doesn't have a tracker, but relies on the DHT to find peers, the trackers can specify tracker URLs for the torrent.

The magnet URI can have trackers specified though...
I'll ask arvid on the mailing list if he has an alternative or if `add_torrent_params::trackers` will do the trick.